src/main.cpp

```diff
@@ -71,6 +72,13 @@ static int inner_main(int argc, char** argv)
     // execution return value
     int retVal = phantom->returnValue();
     delete phantom;
+
+#ifndef NDEBUG
```

Comment:
How about checking QT_DEBUG? I've never liked negatively expressed variables. WTF used NDEBUG where it reported the leak, so that's what I used here.